c7n/resources/ecr.py

```diff
@@ -15,10 +15,18 @@
 
 from botocore.exceptions import ClientError
 
+import json
 from c7n.filters import CrossAccountAccessFilter
 from c7n.manager import resources
 from c7n.query import QueryResourceManager
-from c7n.utils import local_session
+from c7n.actions import ActionRegistry, BaseAction
+from c7n.filters import FilterRegistry
+from c7n.utils import (
+    chunks, local_session, set_annotation, type_schema, dumps)
+from concurrent.futures import as_completed
+
+filters = FilterRegistry('ecr.filters')
```

Comment:
these aren't nesc, the metaclass for resource automatically populates these on the class